org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_manage-groups.scss

```diff
@@ -163,39 +163,15 @@
   }
 
   #DataTables_Table_0 {
-    width: 100% !important;
-    margin-top: 20px;
-
-    th,
-    td {
-      min-width: 100px;
-      line-height: normal !important;
-    }
+    @include mx-expandable-table();
 
-    .DataTables_sort_icon {
-      display: inline-block;
+    .crm-no-children {
+      padding-left: 0;
     }
 
-    .crm-editable-enabled {
-      white-space: normal !important;
-    }
-
-    .crm-group-group_links {
-      > .crm-hover-button,
```

Comment:
All these styling created in a mixing "mx-expandable-table()" that i have included in line #166